README.md

```diff
@@ -34,13 +34,11 @@ generated file by hand.
 
 ## Usage
 
-In a Unix or Node.js command prompt, use:
+Create a WebKit block list ```output.json``` from the Adblock Plus filter list ```input.txt```:
```

Comment:
Please could you wrap the file names in this sentence in a single backtick instead of three?